Tasks/Maven/mavencodeanalysis.ts

```diff
@@ -0,0 +1,231 @@
+/// <reference path="../../definitions/vsts-task-lib.d.ts" />
```

Comment:
See https://code.visualstudio.com/Docs/languages/javascript (TypeScript definition file section) on hints how to stop sprinkling those all over. This might help as well: http://stackoverflow.com/questions/12687779/how-do-you-produce-a-d-ts-typings-definition-file-from-an-existing-javascript